In-spect scanner tweaks

cacogen opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

I saw one on the floor today and the description was unfortunate.

**Does the following:**

- Spellchecks and simplifies the description a bit. Hopefully it reads as well as it can
- Renames it the N-spect scanner, which seemed to me to be a better version of the type of thing the person who added it was going for
- Changes the sound effect from the quaint voiced placeholder-seeming one to something in the same vein I found on Freesound https://freesound.org/people/Tissman/sounds/522328/
- Changes the report to be a child of paper
- Adds encrypted text to the paper

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

Makes it better
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl: cacogen
tweak: In-spect scanner is now called N-spect scanner (the N stands for Nanotrasen)
soundadd: N-spect scanner has a better sound effect
spellcheck: N-spect scanner description reads better
code: N-spect scanner report is now a child of paper
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

cacogen committed:
Spellchecks the in-spect scanner description

- Renames it the N-spect scanner
- Changes the sound effect from the quaint voiced placeholder to something I found on Freesound
- Changes the report to be a piece of paper
- Adds encrypted WGW protocol report to paper

tgstation-server commented:
You currently have a negative Fix/Feature pull request delta of -5. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

cacogen commented:
@ArcaneMusic 

I wasn't aware this was yours until after I made this just now. What do you think?

ArcaneMusic commented:
Seems fine far as I can tell.
Sound replacement is probably for the best (The soul has been CRUSHED underFOOT and is BLEEDING OUT PROFUSELY)
I originally considered making the output paper but I decided against it since it would probably need like an internal paper level system or something for reloading, but that's niche enough and not upheld on anything else, so we can certainly live without it.
Name change and description is good, flavor is upheld and seems need.

God fucking damnit it's just WGW.

cacogen commented on code/modules/cargo/bounties/security.dm:
```diff
@@ -45,7 +45,7 @@
 	name = "Routine Security Inspection"
 	description = "Perform a routine security inspection using an in-spect scanner on the following general area on station:"
```
```suggestion
	description = "Perform a routine security inspection using an N-spect scanner on the following station area:"
```

cacogen committed:
Fixes a mention of in-spect I missed

CameronWoof commented:
"Central Command-issued" would probably read more clearly.

cacogen commented:
Yeah I wasn't sure whether to put a hyphen there or not but I think you're right.

cacogen commented on code/game/objects/items/documents.dm:
```diff
@@ -58,8 +58,9 @@
-	name = "in-spect scanner"
-	desc = "Cental commmand issued inspection device. Does company grade station inspection protocols when activated, and prints encripted sheets of paper regarding the mainenance of the station. Hard to Replace."
+	name = "\improper N-spect scanner"
+	desc = "Central Command issued inspection device. Performs inspections according to Nanotrasen protocols when activated, then \
```
```suggestion
	desc = "Central Command-issued inspection device. Performs inspections according to Nanotrasen protocols when activated, then \
```

cacogen committed:
Central Command-issued

cacogen committed:
No longer uses WGW

- Tweaks desc to perfection
- Uses blank icon for slip without scanned_area set
- Creates report text when scanned_area is set
- Generates a random string to simulate Base64 style encoding

cacogen commented on code/game/objects/items/documents.dm:
```diff
@@ -77,23 +78,40 @@
 	. = ..()
 	if(scanned_area?.name)
-		. += "<span class='notice'>\The [src] contains data on [scanned_area.name].</span>"
+		. += "<span class='notice'>\The [src] contains data on \the [scanned_area.name].</span>"
```
```suggestion
		. += "<span class='notice'>\The [src] contains data on [scanned_area.name].</span>"
```

cacogen committed:
Update code/game/objects/items/documents.dm

cacogen commented on code/game/objects/items/documents.dm:
```diff
@@ -77,23 +78,40 @@
 	var/area/scanned_area
+	show_written_words = FALSE
+
+/obj/item/paper/report/proc/generate_report(var/area/scan_area)
```
```suggestion
/obj/item/paper/report/proc/generate_report(area/scan_area)
```

cacogen committed:
Removes var from proc argument to please cunter

tgstation-server commented:
Your Fix/Feature pull request delta is currently below zero (-3). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

interestingusernam3 commented:
woody
